### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by open-risk**

### CODE REVIEW

**Comment by Purvanshsingh** [comment]
@chrizandr Please do merge this PR.
as it is following #8 (which is already approved) to the main branch

**Review by chrizandr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+|
+|___ APIDOC
+|___ Creditrisk_api_docwriter.py
+```
+`APIDOC` is a pickle serialized object, It can be accessed as follows:
+```python
+import pickle
+
+API_Doc_file = open("creditrisk_poc/API_Doc/APIDOC","rb")
+doc = pickle.load(API_Doc_file)
```
Comment (Mec-iS) [inline README.md:29]: @Purvanshsingh why is this being merged? it is using the `pickle` serialisation that we agreed to change.
#3 should have been merged first and this one only after, please be more attentive

Comment (Mec-iS) [reply README.md:29]: @chrizandr this should have not been approved

Comment (Purvanshsingh) [reply README.md:29]: @Mec-iS  I'll fix that now, it was a small change else everything was approved by phillipos that's why I merged.

Comment (chrizandr) [reply README.md:29]: We had approved this on the call. I thought it was already reviewed by Phillipos as well.

**Pull request merged**